src/components/Books.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import Header from './Header';
+import Bookcard from './Bookcard';
+import Form from './Form';
+
+export default function Books() {
+  return (
+    <div>
+      <Header />
+      <Bookcard title="Dune" author="Frank Berbert" />
+      <hr />
+      <Bookcard title="Capital in the Twenty-First Century" author="Suzanne Collins" />
+      <hr />
+      <Form />
+    </div>
```

Comment:
so close 👍🏾 but this is the only required task
in the book.js component there should be a remove ` button ` .
*NB* this is only a placeholder and should not be able to remove any book. 😸